daemon/src/daemon.rs

```diff
@@ -983,7 +986,12 @@ impl Daemon {
             );
         }
         if app_config.emit_join_code {
-            put_secret_address_into_wormhole(address).await;
+            let address = connection_manager.secret_address().clone();
+            tokio::spawn(async move {
```

Comment:
in WebAssembly we have to use `dioxus::spawn` instead of `tokio::spawn`. it would be helpful to make that distinction in one place and then simply import a general `spawn` from there.